datafusion/src/sql/planner.rs

```diff
@@ -2343,8 +2347,8 @@ mod tests {
                    GROUP BY first_name
                    HAVING MAX(age) > 100 AND MIN(id - 2) < 50";
         let expected = "Projection: #person.first_name, #MAX(person.age)\
-                        \n  Filter: #MAX(person.age) Gt Int64(100) And #MIN(person.id Minus Int64(2)) Lt Int64(50)\
-                        \n    Aggregate: groupBy=[[#person.first_name]], aggr=[[MAX(#person.age), MIN(#person.id Minus Int64(2))]]\
+                        \n  Filter: #MAX(person.age) > Int64(100) AND #MIN(person.id Minus Int64(2)) < Int64(50)\
```

Comment:
@alamb Im a bit confused.  The minus operator seems to be formatted differently depending on the part of the plan its in - is that expected?  The test passes right now but i had to use `Minus` in the `Filter` section and `-` in the `Aggregate` section.

Reply:
One difference is that `MIN` is a aggregate function which is a different type of Expr -- perhaps it formats its arguments using `{:?}`